exovetter/main.py

```diff
@@ -0,0 +1,61 @@
+
+#A sketch of an architecture. For discussion
+
+#Each vetting test is wrapped in a class that follows this
+#structure.
+class Vetter():
```

Comment:
As this is a just a sketch, it's too soon to worry about PEP 8 type things like this. I'd argue by the time the Universe ends it will still be too soon to worry about PEP8 type things, but that's an argument for when we start writing code 😀

Reply:
Usually it is `Vetter:` or `Vetter(object)`, but looks like `Vetter()` works too. TIL.